### PR TITLE
feat(web): arrange site visit

### PR DESCRIPTION
**Pull request by james-hill-kc**
## Describe your changes

arrange site visit

## Issue ticket number and link

BOAT-311

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/web/src/server/appeals/appeal-details/appeal-details.types.d.ts.orig
Comment (fabfab-kc) [inline appeals/web/src/server/appeals/appeal-details/appeal-details.types.d.ts.orig:1]: deleted?

**Review by fabfab-kc — APPROVED** [review]
James just a backup file which shouldn't be there, from my POV...

File: appeals/web/src/server/appeals/appeal-details/appeal-details.types.d.ts.orig
Comment (james-hill-kc) [reply appeals/web/src/server/appeals/appeal-details/appeal-details.types.d.ts.orig:1]: Whoops, must have missed this one, thanks for flagging!

Comment (james-hill-kc) [reply appeals/web/src/server/appeals/appeal-details/appeal-details.types.d.ts.orig:1]: That's been removed now

**Review by fabfab-kc — APPROVED** [review]

**Pull request merged**